ui/articles-aem-renderer/build.gradle

```diff
@@ -5,22 +5,14 @@ android {
 }
 
 dependencies {
-    implementation project(':library:base')
-    implementation project(':library:db')
-    implementation project(':library:download-manager')
-    implementation project(':library:model')
-    implementation project(':library:model-events')
-    implementation project(':library:model-loader')
-    implementation project(':library:sync-tasks')
-
-    implementation project(':ui:base')
 
     implementation "android.arch.lifecycle:extensions:${deps.androidLifecycle}"
     implementation "android.arch.persistence.room:runtime:${deps.roomVersion}"
     implementation "com.android.support:appcompat-v7:${deps.androidSupport}"
 
     androidTestImplementation "com.android.support.test:runner:${deps.androidTestingSupport}"
     androidTestImplementation "android.arch.persistence.room:testing:${deps.roomVersion}"
+    testImplementation "android.arch.core:core-testing:${deps.androidLifecycle}"
```

Comment:
should this have the lifecycle version? or is it a separate architecture component that is versioned independently?

Reply:
also, `testImplementation` targets the unit tests, not android tests. So, this dependency is never used